contrib/epee/include/net/net_ssl.h

```diff
@@ -132,6 +132,7 @@ namespace net_utils
     bool handshake(
       boost::asio::ssl::stream<boost::asio::ip::tcp::socket> &socket,
       boost::asio::ssl::stream_base::handshake_type type,
+      const boost::asio::const_buffer &buffer,
```

Comment:
Drop the `const &`, this is a cheap-to-copy type that can be passed via two registers on most ABIs. Might as well put a default `= {}` too.